src/sbin/lorax-composer

```diff
@@ -149,8 +174,16 @@ if __name__ == '__main__':
     # Import example recipes
     commit_recipe_directory(server.config["GITLOCK"].repo, "master", opts.RECIPES)
 
-    log.info("Starting %s on %s:%d with recipes from %s", VERSION, opts.host, opts.port, opts.RECIPES)
-
-    http_server = WSGIServer((opts.host, opts.port), server, log=LogWrapper(server_log))
+    # Setup the Unix Domain Socket, remove old one, set ownership and permissions
+    if os.path.exists(opts.socket):
+        os.unlink(opts.socket)
+    listener = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+    listener.bind(opts.socket)
+    os.chmod(opts.socket, 0o770)
```

Comment:
Execute permissions are not necessary and confusing, so 0o660 is better.

Reply:
Yeah, not sure why I did that.